roboteam_ai/src/control/ControlUtils.cpp

```diff
@@ -346,7 +346,7 @@ const world::World::RobotPtr ControlUtils::getRobotClosestToLine(std::vector<wor
 }
 
     Vector2 ControlUtils::getInterceptPointOnLegalPosition(Vector2 position, Line line, bool canMoveInDefenseArea, bool canMoveOutOfField, double defenseAreamargin, double outOfFieldMargin) {
-        LineSegment shotLine(line.start, line.end + line.end + (line.end - line.start) * 10000);
+        LineSegment shotLine(line.start, line.end + (line.end - line.start).stretchToLength(100.0));
```

Comment:
100 times is not enough in case of short distances.

Reply:
read: stretchToLength. the field is smaller than 100m so this always works.